analyzers/tests/SonarAnalyzer.UnitTest/Rules/ConditionalSimplificationTest.cs

```diff
@@ -30,78 +28,63 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class ConditionalSimplificationTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<ConditionalSimplification>();
+        private readonly VerifierBuilder codeFix = new VerifierBuilder<ConditionalSimplification>().WithCodeFix<ConditionalSimplificationCodeFix>();
+
         [TestMethod]
         public void ConditionalSimplification_BeforeCSharp8() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\ConditionalSimplification.BeforeCSharp8.cs",
-                                    new ConditionalSimplification(),
-                                    ParseOptionsHelper.BeforeCSharp8);
+            builder.AddPaths("ConditionalSimplification.BeforeCSharp8.cs").WithOptions(ParseOptionsHelper.BeforeCSharp8).Verify();
 
         [TestMethod]
         public void ConditionalSimplification_CSharp8() =>
-            new VerifierBuilder<ConditionalSimplification>()
-                .AddPaths("ConditionalSimplification.CSharp8.cs")
-                .WithLanguageVersion(LanguageVersion.CSharp8)
-                .Verify();
+            builder.AddPaths("ConditionalSimplification.CSharp8.cs").WithLanguageVersion(LanguageVersion.CSharp8).Verify();
 
         [TestMethod]
         public void ConditionalSimplification_CSharp8_CodeFix() =>
-            OldVerifier.VerifyCodeFix<ConditionalSimplificationCodeFix>(
-                @"TestCases\ConditionalSimplification.CSharp8.cs",
-                @"TestCases\ConditionalSimplification.CSharp8.Fixed.cs",
-                new ConditionalSimplification(),
-                ImmutableArray.Create<ParseOptions>(new CSharpParseOptions(LanguageVersion.CSharp8)));  // ToDo: Use WithLanguageVersion instead
+            codeFix.AddPaths("ConditionalSimplification.CSharp8.cs").WithLanguageVersion(LanguageVersion.CSharp8).WithCodeFixedPath("ConditionalSimplification.CSharp8.Fixed.cs").VerifyCodeFix();
 
         [TestMethod]
         public void ConditionalSimplification_FromCSharp8() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\ConditionalSimplification.FromCSharp8.cs",
-                                    new ConditionalSimplification(),
-                                    ParseOptionsHelper.FromCSharp8);
-#if NET
-        [TestMethod]
-        public void ConditionalSimplification_FromCSharp9() =>
-            OldVerifier.VerifyAnalyzerFromCSharp9Console(@"TestCases\ConditionalSimplification.FromCSharp9.cs",
-                                                      new ConditionalSimplification());
-
-        [TestMethod]
-        public void ConditionalSimplification_FromCSharp10() =>
-            OldVerifier.VerifyAnalyzerFromCSharp10Console(@"TestCases\ConditionalSimplification.FromCSharp10.cs",
-                                                       new ConditionalSimplification());
-#endif
+            builder.AddPaths("ConditionalSimplification.FromCSharp8.cs").WithOptions(ParseOptionsHelper.FromCSharp8).Verify();
```

Comment:
I prefer the single line where possible. This is a complicated case, in most of the cases, it will look way better as a single line. For example here https://github.com/SonarSource/sonar-dotnet/blob/Pavel/OldVerifier-CSharp10/analyzers/tests/SonarAnalyzer.UnitTest/Rules/MutableFieldsShouldNotBePublicStaticTest.cs